buildconfig/CMake/Bootstrap.cmake

```diff
@@ -10,7 +10,7 @@ if( MSVC )
   include ( ExternalProject )
   set( EXTERNAL_ROOT ${PROJECT_SOURCE_DIR}/external )
   set( THIRD_PARTY_GIT_URL "https://github.com/mantidproject/thirdparty-msvc2015.git" )
-  set ( THIRD_PARTY_GIT_SHA1 4691e740fa2050f4fd0b44628eac445906bb2978 )
+  set ( THIRD_PARTY_GIT_SHA1 ac57236e560ef78a60cdec32503c26cf013cdf08 )
```

Comment:
@matthew-d-jones I've updated the Windows Kafka libraries to remove the LZ4 dependency. Can you update the SHA1 here to `09789e5ed0309f94b23eb4c8fa5e76ba07dc262b`.